ci-operator/config/openshift/machine-config-operator/openshift-machine-config-operator-master.yaml

```diff
@@ -18,6 +18,9 @@ images:
 - dockerfile_path: Dockerfile.machine-config-operator
   from: base
   to: machine-config-operator
+- dockerfile_path: Dockerfile.setup-etcd-environment
+  from: base
+  to: setup-etcd-environment
```

Comment:
Could we pick a name that scopes this into the usage that it will have - like `installer-bootstrap-etcd`?